.github/workflows/ui-components.yml

```diff
@@ -31,4 +31,6 @@ jobs:
         run: yarn ui-components build
       - uses: JS-DevTools/npm-publish@v1
         with:
+          package: './packages/ui-components/package.json'
```

Comment:
We should also add the version, and release on npm only tags

Reply:
As version it use as default the one in the `package.json`, while this action release on npm only when the version in `package.json` differ from the latest one on NPM

Reply:
![image](https://user-images.githubusercontent.com/6383527/118792304-d353f780-b897-11eb-9643-735ef7cf16e6.png)